proj/src/kmeans.cpp

```diff
@@ -289,3 +297,128 @@ std::vector<int> argmin_omp(std::vector<std::vector<double>> &vec, int nthreads)
     return min_indices;
 }
 
+
+
+#ifdef WCUDA

```

Comment:
You probably may want to know https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#nvcc-identification-macro